components/form/sign-up-form.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
+import { useFormik, Field } from 'formik'
+import { Button } from '../data-components/button'
+import { PageType } from '../types'
+
+function validateForm(values: {
+  firstName: any
+  lastName: any
+  email: any
+  phone?: string
+  text?: string
+}) {
+  let errors: { [k: string]: any } = {}
```

Comment:
The errors in the form should be localized. So we probably need to add those to the CMS. let me know what you think.